### PR TITLE
Language Changes

### PR DESCRIPTION
**Pull request by jme783**
- Particle Cloud --> Particle Device Cloud
- System Firmware --> Device OS

These changes reflect our new product marketing / branding efforts at Particle.

### CODE REVIEW

File: src/content/guide/how-to-build-a-product/manufacturing.md [hunk 1]
```diff
@@ -15,7 +15,20 @@ In most cases, manufacturing with Particle hardware is the same as manufacturing
...
+- **Purchasing from Particle** means that you're buying either Photons
+or PØ/P1 modules through our [online store](http://store.particle.io),
```
Comment (anatoly-particle) [inline src/content/guide/how-to-build-a-product/manufacturing.md:19]: Why do yo wrap lines in some places at 70 characters?

**Review by anatoly-particle — APPROVED** [review]
👍 
Btw, do I understand it correctly:
— “Particle cloud” is a part of backend architecture
— while the “Particle Cloud” is a proper noun name of the product?

File: src/content/guide/how-to-build-a-product/manufacturing.md (hunk 1 above)
Comment (jme783) [reply src/content/guide/how-to-build-a-product/manufacturing.md:19]: My editor's default for .md files :)

Comment (anatoly-particle) [reply src/content/guide/how-to-build-a-product/manufacturing.md:19]: @jme783 https://github.com/particle-iot/docs/pull/779 😉
My bad: missed it.

**Comment by jme783** [comment]
@anatoly-particle — externally to our community, we should refer to the broad set of software tools and services we provide as part of the "Particle Device Cloud"

**Pull request merged**